notifications_utils/polygons.py

```diff
@@ -45,15 +67,91 @@ class Polygons():
 
     output_precision_in_decimal_places = 6
 
-    def __init__(self, polygons):
-        if not polygons:
-            self.polygons = []
-        elif isinstance(polygons[0], list):
-            self.polygons = [
-                Polygon(polygon) for polygon in polygons
-            ]
+    def __init__(self, polygons, utm_crs=None):
```

Comment:
Just coming to this as part of reviewing a follow-up PR. Us in the future won't (easily) have the benefit of the PR description to work out what "utm_crs" and "utm_polygons" means. We should add a comment on this and the `utm_polygons` method to explain what they're doing:

- The input data may be the WSG84 coordinates (would really help to mention "traditional lat/lon degrees" here) based on what we store or UTM ("cartesian plane") coordinates in metres - it's hard to find [a reference for metres](https://pyproj4.github.io/pyproj/stable/api/transformer.html#pyproj-transform).

- We use the projected UTM coordinates in memory to perform calculations on polygons in terms of metres. The Shapely library also requires coordinates are in a cartesian plane to perform accurate transformations.

Reply:
Added in https://github.com/alphagov/notifications-utils/pull/892